README.md

```diff
@@ -88,25 +88,23 @@ elements do not include any attributes.
 [Attribute policies](https://static.javadoc.io/com.googlecode.owasp-java-html-sanitizer/owasp-java-html-sanitizer/20220608.1/org/owasp/html/AttributePolicy.html) allow running custom code too.  Adding an attribute policy will not water down any default policy like `style` or URL attribute checks.
 
 ```Java
-new HtmlPolicyBuilder = new HtmlPolicyBuilder()
+PolicyFactory myPolicy = new HtmlPolicyBuilder()
```

Comment:
Maybe `HtmlSanitizer.Policy myPolicy = ...` and keep the `.build()` below.

Reply:
I made it consistent with the first two examples in README

Reply:
Btw. there is no `build()` one build with further parameters...

Reply:
Ah.  Right.  build(...) is when we connect to an output channel and change listener.

Reply:
So, what's the status of thie PR?